tgui/packages/tgui/interfaces/MessageModal.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author ZeWaka (https://github.com/zewaka)
+ * @license MIT
+ */
+
+import { Loader } from './common/Loader';
+import { useBackend } from '../backend';
+import { Box, Section } from '../components';
+import { Window } from '../layouts';
+import { KEY_ESCAPE } from 'common/keycodes';
+import { sanitizeDefAllowTags, sanitizeText } from '../sanitize';
+
+type MessageInputData = {
+  message: string;
+  timeout: number;
+  title: string;
+  theme: string;
+  sanitize: boolean;
+};
+
+export const MessageModal = (_, context) => {
+  const { act, data } = useBackend<MessageInputData>(context);
+  const { message, timeout, title, theme, sanitize } = data;
+  // Dynamically changes the window height based on the message.
+  const windowHeight
+    = 125 + Math.ceil(message?.length / 3);
+
+  let outputMessage = message;
+  if (sanitize) {
+    let allowedHTMLTags = sanitizeDefAllowTags;
+    allowedHTMLTags.push('a'); // We commonly want to let users redirect to a URL.
```

Comment:
This will mutate the imported array. Not worth me trying to work out if that's okay in this particular stack of transpiled module bundlers. Try the below to work regardless:
```
const allowedHTMLTags = [...sanitizeDefAllowTags, 'a'];
```
In either case, future tgui project to have a Markdown renderer. 😀